LunaBot/Database/User.cs

```diff
@@ -41,6 +41,10 @@ public class User
 
         public bool Monk { get; set; }
 
+        public int SnugR { get; set; }
+
+        public int SnugG { get; set; }
```

Comment:
Make sure you set these variables when you register a user. Because right now it's gonna be null and will throw an exception. 
Register user:
https://github.com/javierdlg/LunaBot/blob/fb96d1a53a7ea91e360cda19932bd5b1948c1565/LunaBot/Commands/User/RegisterCommand.cs#L44